dcpy/connectors/edm/recipes.py

```diff
@@ -245,6 +248,11 @@ def purge_recipe_cache():
 
 
 def log_metadata(config: library.Config):
+    # long term, it probably makes more sense to have db interactions isolated by environment as well
```

Comment:
oh I see.

Reply:
Btw, are we running `log_metadata` when using `ingest`, or only under `library`?

Reply:
Currently `library`, though I should change that. About to work more on `ingest` so I'll incorporate logging there